src/calendar/index.js

```diff
@@ -158,6 +165,13 @@ class Calendar extends Component {
     this.updateMonth(this.state.currentMonth.clone().addMonths(count, true));
   }
 
+  intzAccessibilityLabel(state, day) {
+    const today = XDate.locales[XDate.defaultLocale].today;
+    const ftAccesibilitylabel = XDate.locales[XDate.defaultLocale].ftAccesibilitylabel;
+    if (ftAccesibilitylabel) return `${state === 'today' ? today : ''} ${day.toString(ftAccesibilitylabel)} ${this.getMarkingLabel(day)}`;
+    return `${state === 'today' ? 'today' : ''} ${day.toString('dddd d MMMM yyyy')} ${this.getMarkingLabel(day)}`;
```

Comment:
are you returning the string Today? we should support locale.

Reply:
Currently the library does not support a format for accessibility, it only takes English, so the key ftAccesibilitylabel was added, I check if this key is declared and if it is not left the one by default, basically I leave everything as it was, just add that of the format of accessibility for blind people